test/spec.yml

```diff
@@ -297,6 +297,7 @@
         out: |-
             Created function * in /home/dockeruser/test
               (use "bn deploy gulliblezebra" to deploy the function)
+    -   in: sed -i '2 a \  console.log(`Hello ${name}!`);' function.js
```

Comment:
I would replace this with 
```
    -   in: |-
           cat > function.js << EOL
           exports.handler = async (body, req) => { console.log(`Hello ${name}!`) };
           EOL
```

Far more readable.

Reply:
done. please have a second look.